modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/provider/LayoutInfoItemLanguagesProvider.java

```diff
@@ -52,6 +52,11 @@ public String getDefaultLanguageId(Layout layout) {
 			layout);
 	}
 
+	@Override
+	public String[] getInfoItemAvailableLanguageIds(Layout layout) {
+		return layout.getAvailableLanguageIds();
+	}
+
```

Comment:
Cannot we get rid of the other logic and simply return all layout languages? Shouldn't the layout include all the languages for all its fragments? Maybe the original implementation is not really needed.

Reply:
Maybe this way we no longer need to change the interface, and the code will be simpler too.

Reply:
The problem is what we decide to be translated or not depending on the type of page.

We are using the old method for two things 

- translate the page
- add the meta tag for the html 


Changing this behavior will rise more issues. 


That is why I added a new method. I was tempted to add a parameter but it would break the homogeneity of the api